package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "build": "tsc",
+    "build-examples": "tsc && cd ./examples && npm install && rm -r ./node_modules/revai-node-sdk/src && cp -r ../dist/src ./node_modules/revai-node-sdk",
```

Comment:
The `cd ./examples` is a little suspicious. It will actually change where to user's terminal is focused, no?

Reply:
Might be wrong but I believe the npm script can only be run from the project root directory (where `package.json` lives) - so `cd ./examples` will be in respect to the project root directory - if it was `cd examples` it may be more suspicious?